src/integration/pages/articles/news/__snapshots__/canonical.test.js.snap

```diff
@@ -237,6 +237,8 @@ exports[`Canonical Articles Most Read has item with rank 10`] = `
 }
 `;
 
+exports[`Canonical Articles Paragraph should match paragraph text 1`] = `"Image source, St Joseph's Hospice"`;
```

Comment:
This seems to be an image caption.. which is probably OK in this scenario?

Reply:
changing the tests to make sure they check only body text paragraphs